packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -147,6 +149,84 @@ class InteractiveViewer extends StatefulWidget {
        constrained = false,
        child = null;
 
+  /// When laying out the child InteractiveViewer applies a transform based on the current
```

Comment:
Does this first phrase need a comma after "child" ?

Reply:
Fixed!